protos/fpc/fpc.proto

```diff
@@ -66,3 +66,36 @@ message Credentials {
     // serialized attestation evidence as output by `AttestationToEvidence`, see `interfaces.attestation.md`
     bytes evidence = 3;
 }
+
+message ChaincodeRequest {
+    // function to be invoked on FPC chaincode
+    string function = 1;
+
+    // function arguments of above
+    repeated string arguments = 2;
+
+    // key to (aes128-gcm) encrypt ChaincodeResponse
+    bytes returnEncryptionKey = 3;
```

Comment:
```suggestion
    bytes response_encryption_key = 3;
```

Reply:
https://developers.google.com/protocol-buffers/docs/style

We do not follow the style guide entirely in this file :/ For instance: `message Host_Parameters` We can revisit this later.

Reply:
i noticed also that Fabric is also slightly inconsistent. Have to admit i haven't seen the link you mentioned but reading it now it seems kindof important to follow the guide so the mapping is consistent across languates:
=>
If we ever want to make it consistent, it's probably better to do that sooner than later: should i do that already as part of this PR?

Reply:
Ok, after the offline discussion, i've added the corresponding changes as a separate commit ...